tokio 1.16.1 doesn't compile for mipsel

**Description**

Failed at https://github.com/tokio-rs/tokio/blob/tokio-1.16.1/tokio/src/runtime/metrics/worker.rs#L2

```
   Compiling tokio v1.16.1
error[E0432]: unresolved import `std::sync::atomic::AtomicU64`
Error:  --> /cargo/registry/src/github.com-1ecc6299db9ec823/tokio-1.16.1/src/runtime/metrics/worker.rs:2:25
  |
2 | use std::sync::atomic::{AtomicU64, AtomicUsize};
  |                         ^^^^^^^^^
  |                         |
  |                         no `AtomicU64` in `sync::atomic`
  |                         help: a similar name exists in the module: `AtomicU8`

error[E0599]: no method named `fetch_add` found for struct `AtomicU64` in the current scope
Error:   --> /cargo/registry/src/github.com-1ecc6299db9ec823/tokio-1.16.1/src/runtime/metrics/scheduler.rs:25:36
   |
25 |         self.remote_schedule_count.fetch_add(1, Relaxed);
   |                                    ^^^^^^^^^ help: there is an associated function with a similar name: `fetch_or`
   | 
  ::: /cargo/registry/src/github.com-1ecc6299db9ec823/tokio-1.16.1/src/loom/std/atomic_u64.rs:18:5
   |
18 |     pub(crate) struct AtomicU64 {
   |     --------------------------- method `fetch_add` not found for this

```

[Detailed log](https://github.com/rabbit-digger/rabbit-digger-pro/runs/5059231896?check_suite_focus=true#step:5:265)

It compiled after I downgraded tokio to [`1.15.0`](https://github.com/rabbit-digger/rabbit-digger-pro/runs/5059723952?check_suite_focus=true#step:5:262). And I guess it's not fixed in #4453 


**Related**

https://github.com/tokio-rs/tokio/issues/4452 https://github.com/tokio-rs/tokio/issues/4454 https://github.com/tokio-rs/tokio/pull/4453

--- a/tokio/src/macros/cfg.rs
+++ b/tokio/src/macros/cfg.rs
@@ -177,9 +177,9 @@ macro_rules! cfg_macros {
 macro_rules! cfg_metrics {
     ($($item:item)*) => {
         $(
-            // For now, metrics is always enabled. When stabilized, it might
-            // have a dedicated feature flag.
-            #[cfg(tokio_unstable)]
+            // For now, metrics is only disabled in loom tests.
+            // When stabilized, it might have a dedicated feature flag.
+            #[cfg(all(tokio_unstable, not(loom)))]
             #[cfg_attr(docsrs, doc(cfg(tokio_unstable)))]
             $item
         )*
@@ -189,7 +189,7 @@ macro_rules! cfg_metrics {
 macro_rules! cfg_not_metrics {
     ($($item:item)*) => {
         $(
-            #[cfg(not(tokio_unstable))]
+            #[cfg(not(all(tokio_unstable, not(loom))))]
             $item
         )*
     }

--- a/tokio/src/runtime/metrics/worker.rs
+++ b/tokio/src/runtime/metrics/worker.rs
@@ -1,5 +1,5 @@
-use std::sync::atomic::Ordering::Relaxed;
-use std::sync::atomic::{AtomicU64, AtomicUsize};
+use crate::loom::sync::atomic::Ordering::Relaxed;
+use crate::loom::sync::atomic::{AtomicU64, AtomicUsize};
 
 /// Retreive runtime worker metrics.
 ///

